build.cake

```diff
@@ -57,6 +58,7 @@ var artifactFolder = $"{workingDirectory}/{buildPlan.ArtifactsFolder}";
 var publishFolder = $"{artifactFolder}/publish";
 var logFolder = $"{artifactFolder}/logs";
 var packageFolder = $"{artifactFolder}/package";
+var scriptFolder = $"{workingDirectory}/scripts";
```

Comment:
how about move this into artifacts folder. this keeps the convention that all content under artifacts are disposable.